grc/difi_sink_cpp.block.yml

```diff
@@ -51,35 +51,19 @@ parameters:
 - id: reference_time_full
   label: Reference Time Full
   dtype: int
-  default: 0
+  default: 1666209080
 - id: reference_time_frac
   label: Reference Time Frac
   dtype: int
-  default: 0
+  default: 964925050735
 - id: data_size
   label: Samples Per Packet
   dtype: int
   default: 128
 - id: stream_id
   label: Stream ID
   dtype: int
-  default: 0
-  hide: ${ ('all' if mode == 'True' else 'none') }
-- id: reference_point
-  label: Reference Point
-  dtype: enum
-  options: ['352', '101']
-  option_labels: ['ADC', 'DAC']
-  hide: ${ ('all' if mode == 'True' else 'none') }
-- id: packet_class
-  label: Packet Class
-  dtype: enum
-  options: ['0', '1', '2']
-  hide: ${ ('all' if mode == 'True' else 'none') }
-- id: oui
-  label: Orginization Unique ID
-  dtype: int
-  default: 0
+  default: 80000
```

Comment:
Do you think we should also make the DIFI Source port 80000 by default so they match?

Reply:
also this may have already been the case before your changes but in Paired mode it looks like the last 3 block params are hidden in GRC for some reason

Reply:
Yes, matching default stream IDs would be better. (Done.)

In paired mode, packets received by the source block
should override some sink block parameters.